autogluon/utils/tabular/features/generators/category.py

```diff
@@ -124,3 +124,13 @@ def _generate_category_map(self, X: DataFrame) -> (DataFrame, dict):
             return X_category, category_map
         else:
             return DataFrame(index=X.index), None
+
+    def _remove_features_in(self, features: list):
+        super()._remove_features_in(features)
+        if self.category_map:
+            for feature in features:
+                if feature in self.category_map:
+                    self.category_map.pop(feature)
+
+    def _more_tags(self):
+        return {'feature_interactions': False}
```

Comment:
many newlines missing at bottom of files

Reply:
good catch, added newlines